specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2021-10-30-preview/definitions/Common.json

```diff
@@ -291,6 +291,22 @@
           "type": "string",
           "description": "Data source in the format Protocol:MachineName\\SQLServerInstanceName,PortNumber"
         },
+        "serverName": {
+          "type": "string",
+          "description": "name of the server"
+        },
+        "serverBrandVersion": {
```

Comment:
Didn't see serverBrandVersion flagged in s360 report for SqlConnectionInfo object

Reply:
> Didn't see serverBrandVersion flagged in s360 report for SqlConnectionInfo object

@ash1625 thanks, removed now